compiler/luci-interpreter/src/kernels/If.test.cpp

```diff
@@ -88,9 +88,9 @@ TEST(IfTest, CondTrue)
   EXPECT_THAT(extractTensorData<float>(output), ElementsAreArray(ArrayFloatNear({6, 9})));
 }
 
-TEST(IfTest, CondFalse)
```

Comment:
It seems that this test (`CondFalse`) is removed. Is it intended?

Reply:
Oh, That's my mistake. I will fix it.